server/controllers/apply/applicationsController.test.ts

```diff
@@ -39,12 +51,95 @@ describe('applicationsController', () => {
 
       requestHandler(request, response, next)
 
-      expect(response.render).toHaveBeenCalledWith('applications/new', {
+      expect(response.render).toHaveBeenCalledWith('applications/start', {
         pageHeading: 'Apply for an Approved Premises (AP) placement',
       })
     })
   })
 
+  describe('new', () => {
+    describe('If there is a CRN in the flash', () => {
+      it('it should render the start of the application form', async () => {
+        const person = personFactory.build()
+        personService.findByCrn.mockResolvedValue(person)
```

Comment:
Might make sense to have this in a `beforeEach` step, as we have in the booking test now

Reply:
Absolutely!